core/internal/runconfig/runconfig.go

```diff
@@ -10,7 +10,7 @@ import (
 //
 // This is a type alias for refactoring purposes; it should be new type
 // otherwise.
-type RunConfigDict = pathtree.TreeData
+type RunConfigTree = pathtree.TreeData
```

Comment:
Looks like this alias isn't needed anymore, since the `map[string]any` type is encapsulated by PathTree.

Reply:
Yep… could be replaces